code/core/src/manager/globals/runtime.ts

```diff
@@ -38,6 +39,7 @@ export const globalsNameValueMap: Required<Record<keyof typeof globalsNameRefere
   'storybook/internal/components': COMPONENTS,
   'storybook/internal/core-events': EVENTS,
   'storybook/internal/manager-errors': EVENTS_MANAGER_ERRORS,
+  'storybook/internal/manager/manager-stores': MANAGER_STORES,
```

Comment:
What are manager-stores? The `code/core/src/manager/manager-stores.ts`-file seems to have existed before, why is this extra globalization required now?